fizzBuzzRedux.js

```diff
@@ -0,0 +1,51 @@
+//Benton Green
+  function FizzBuzz(input1, input2){
+    var startNum;
+    var endNum;
+    this.input1 = input1;
+    this.input2 = input2;
+    
+    this.readFizzBuzz = function(startNum, endNum){
+      startingNum = startNum;
+      endingNum = endNum;
+    };
+
+    this.writeFizzBuzz = function() {
```

Comment:
Since this never gets called by a public method, you may want to refactor it to be a private function (`function writeFizzBuzz`) and then call it from your public method `this.writeToFizzBuzz`). For _this_ project, I'd probably just merge the two, but if you were doing something more complex and wanted to break it in to steps, that would be the way to go.